src/index.js

```diff
@@ -0,0 +1,7 @@
+import './style.css';
+
+document.getElementById('recentScores').innerHTML = `<ul class="scoreList">
+<li>Name: 100</li>
+<li>Name: 200</li>
+<li>Name: 50</li>
+</ul>`;
```

Comment:
- [ ] good job implementing javascript into your project 👍  however in the requirements and in the previous review it is specifically requested to use ES6 modules, which includes usage of arrow functions, usage of import and export instead of `require` or `module.exports` and proper usage of const and let keywords.
Please have another look at the lessons regarding [ES6 modules](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/lessons/lesson_es6_modules.md)